src/plugins/three/objects/EllipsoidRegionHelper.js

```diff
@@ -81,8 +81,12 @@ function getRegionGeometry( ellipsoidRegion ) {
 
 	}
 
-	// compute the vertex normals so we can get the edge normals
-	geometry.computeVertexNormals();
+	if ( computeNormals ) {
+
+		// compute the vertex normals so we can get the edge normals
+		geometry.computeVertexNormals();
+
+	}
```

Comment:
Can you explain this change?

Reply:
When profiling a tileset with region bounds, I noticed that the biggest performance hit was computing normals for ellipsoid helpers. However, the line-based variant of the helper do not need normals AFAIK.